### DISCUSSION TITLE
Running for changing "static" features

### DISCUSSION THREAD

**Discussion by Flash-Of-Thunder** [neuralhydrology/neuralhydrology#53] [Q&A] [answered]
Hello, l was looking for general guidance at first on trying to run models with changing "static" features. E.g. I want to quantify and predict the impact of land development on streamflows. Let's say I have measurements of % impervious area, % forest,  over 1980 - 2010 in rapidly developing basins, so these numbers change year to year. Could they be modeled with this package, would they actually be entered with "meteorological forcing" data?

**Comment by kratzert** [comment, marked as answer]
Hi there,
yes that is possible by e.g. using the `evolving_attributes` config argument. Check the [documentation](https://neuralhydrology.readthedocs.io/en/latest/usage/config.html#data-settings) for details and make sure to pass these features via the additional feature files argument (although I think it should also work if they are just a column in your normal time series data file).

These features will then be used inside of the model as static inputs (some models have specific use cases for these inputs while others just copy them and concatenate them at every time step to the dynamic inputs, e.g. precipitation).

Other option, as you said, is to simply use them as another feature in the `dynamic_inputs` list. It really depends if you want to use one value per each of this feature for a single input sequence (then use evolving attributes) or if you want this feature to be dynamic although within a single input sequence (then use dynamic inputs argument).

Let me know if you have a follow up question but note that I might need some time to answer, as I am in parental leave.

**Reply by Flash-Of-Thunder** [reply]
Thanks! I'll test this out. Congrats on the baby!
